src/Storage/Commands.Storage.Test/Common/Cmdlet/NewAzureStorageContextTest.cs

```diff
@@ -20,9 +20,74 @@
 using Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet;
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Auth;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using System.Collections.Generic;
 
 namespace Microsoft.WindowsAzure.Commands.Storage.Test.Common.Cmdlet
 {
+    class TestProfileProvider : AzureRmProfileProvider
```

Comment:
@blueww Please add any additional scenarios to the tests I added, as xunit tests.  We are not accepting any new MSTest tests in the repo, and these need to be check-in tests